nakama/nakama.lua

```diff
@@ -4179,18 +2682,13 @@ end
 --- rpc_func2
 -- Execute a Lua function on the server.
 -- @param client Nakama client.
--- @param id_str (string) The identifier of the function..
--- @param payload_str (string) The payload of the function which must be a JSON object..
--- @param http_key_str (string) The authentication key used when executed as a non-client HTTP request..
+-- @param id_str (string) The identifier of the function.
+-- @param payload_str (string) The payload of the function which must be a JSON object.
+-- @param http_key_str (string) The authentication key used when executed as a non-client HTTP request.
 -- @param callback Optional callback function.
 -- A coroutine is used and the result returned if no function is provided.
 -- @return The result.
-function M.rpc_func2(
-	client
-	,id_str
-	,payload_str
-	,http_key_str
-	,callback)
+function M.rpc_func2(client, id_str, payload_str, http_key_str,callback)
```

Comment:
Instead of having an `rpc_func2` that can take a payload, can we remove it and just let `rpc_func` accept a payload?

Reply:
Both of these are defined in apigrpc.swagger.json which means that both will get generated. I can add some special code to remove one or the other.